packages/@okta/vuepress-site/docs/guides/request-user-consent/main/index.md

```diff
@@ -21,36 +19,37 @@ Implement an Okta-hosted user consent dialog box.
+You can configure which scopes aren't required, which are optional, and which are required.
+
+When an application needs to get a new access token from an authorization server, the user isn't prompted for consent if they already consented to the specified scopes. Consent grants remain valid until the user or admin manually revokes them, or until the user, application, authorization server, or scope is deactivated or deleted.
 
 > **Note:** The user only has to grant consent once for an attribute per authorization server.
 
 When a consent dialog appears depends on the values of three elements:
 
 * `prompt`: a query [parameter](/docs/reference/api/oidc/#parameter-details) that is used in requests to `/oauth2/${authorizationServerId}/v1/authorize` (custom authorization server)
-* `consent_method`: a property listed in the **Settings** [table](/docs/reference/api/apps/#settings-10) in the Apps API doc
-* `consent`: a property listed in the **Parameter details** [section](/docs/reference/api/oidc/#parameter-details) for the `/authorize` endpoint
+* `consent_method`: an application property listed in the **Settings** [table](/docs/reference/api/apps/#settings-10) in the Apps API doc. This property allows you to determine whether a client is fully trusted (for example, a first-party application) or requires consent (for example, a third-party application).
+* `consent`: a scope property listed in the **Parameter details** [section](/docs/reference/api/oidc/#parameter-details) for the `/authorize` endpoint. This property allows you to enable or disable user consent for an individual scope.
 
 ## Enable consent for scopes
 
-Use the following steps to display the user consent dialog box as part of an OpenID Connect or OAuth 2.0 request.
+Use the following steps to display the user consent dialog as part of an OpenID Connect or OAuth 2.0 request.
 
 > **Note:** Currently OAuth Consent works only with custom authorization servers. See [Authorization Servers](/docs/concepts/auth-servers/) for more information on the types of authorization servers available to you and what you can use them for.
 
 <ApiAmProdWarning />
 
 1. In the Admin Console, go to **Applications** > **Applications**.
 1. Select the OpenID Connect app that you want to require user consent for.
-1. On the **General** tab, in the **General Settings** section, click **Edit**.
-1. Scroll down to the **User Consent** section and select **Require consent**.
+1. On the **General** tab, scroll down to the **User Consent** section and verify that the **Require consent** checkbox is selected. If it isn't, click **Edit** and select **Require consent**.
 
     > **Note:** If the **User Consent** section doesn't appear, you don't have the API Access Management and the User Consent features enabled. To enable these features, contact [Support](https://support.okta.com/help/open_case?_).
```

Comment:
There is a white space in the contact hyperlink.

Reply:
Where?